_drafts/oryx/reactivity/key-concepts-of-reactivity.md

```diff
@@ -0,0 +1,84 @@
+- Ability to emit more than one value.
+- Ability to resolve value both asynchronously and synchronously.
+- Ability to utilize tried and tested operators to combine streams and define reactive logic.
+
+## Application state
+
+Application state is data that describes the _current_ state of an application at any given moment. It is used to render the user interface and provide the right behavior for user interactions.
+
+Oryx does not provide a _global_ state management layer. Application state is maintained per _domain_, and each domain is only concerned with the associated data used by the domain. For example, product data is maintained in the product domain, and the product domain logic ensures that a product is only loaded once when multiple requests are being made for the same product.
+
+Some part of the application state is maintained below domains. A good example is the internationalization state (or application context), like active language and currency. The internationalization domain is considered a core domain, so it can be used in other domains without introducing cycle dependencies in the system.
+
+## Application layers
+
+Oryx simplifies working with asynchronous application state and reactivity by handling the complexity under the hood while supporting customization. It provides vanilla JavaScript packages for services and lower-level layers, which can be reused by developers regardless of UI framework.
+
+Most of the application state is driven by loading data from backend APIs. Oryx provides the following standardized application layers:
+
+| LAYER      | PURPOSE                                                                                                                                                                                                                                                                                                                                  |
+| ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Component  | Renders application state inside UI elements.                                                                                                                                                                                                                                                                                            |
+| Controller | Resolves application state for a given context, so that the component can be reused in different places. For example, a product title component can be used on the Product Details Page, product card, and in a cart entry. The controller resolves the product _sku_ from the context, so that the right product title can be resolved. |
```

Comment:
```suggestion
| Controller | Resolves application state for a given context, so that the component can be reused in different places. For example, a product title component can be used on the Product Details Page, product card, and in a cart entry. The controller resolves the product _SKU_ from the context so that the right product title can be resolved. |
```